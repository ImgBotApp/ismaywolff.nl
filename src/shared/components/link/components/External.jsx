import Internal from './Internal'

const External = Internal.withComponent('a')

export default External
