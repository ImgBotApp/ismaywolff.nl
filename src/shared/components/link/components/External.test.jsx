/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import External from './External'

describe('<External />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<External href="/url">Text</External>)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })

  it('renders clean correctly', () => {
    const wrapper = shallow(<External href="/url" clean>Text</External>)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })

  it('renders size correctly', () => {
    const wrapper = shallow(<External href="/url" size="16px">Text</External>)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })
})
