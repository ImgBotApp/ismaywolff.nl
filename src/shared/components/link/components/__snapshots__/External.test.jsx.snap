// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<External /> renders clean correctly 1`] = `
.hllEJG {
  color: currentColor;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  className="sc-bwzfXH hllEJG"
  href="/url"
>
  Text
</a>
`;

exports[`<External /> renders correctly 1`] = `
.hRLKqa {
  color: currentColor;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<a
  className="sc-bwzfXH hRLKqa"
  href="/url"
>
  Text
</a>
`;
